src/app.js

```diff
@@ -74,7 +92,7 @@ export function app(app) {
 
   function render(state, view) {
     element = patch(
-      app.root || (app.root = document.body),
+      app.root,
```

Comment:
Will be handled in load() won't it?  I moved that to load so that I wasn't duplicating that logic